frontend/src/components/manage-cookies.tsx

```diff
@@ -15,13 +14,7 @@ export const ManageCookiesLink = () => {
         osano.cm.showDrawer('osano-cm-dom-info-dialog-open')
     }
 
-
     return (
-        <>
-            <Global
```

Comment:
moved to main.scss

Reply:
Hm, I slightly prefer it here because this way we only hide the osano cookie link if we're also rendering an alternative method.  If the hiding is in the sass, it's possible that we hide it, but then don't render this on some screens

Reply:
I should have added another comment for context woops, we moved the manage cookies link into the user name/dropdown thing at the top right, with that being there it's not technically rendered until the dropdown is opened. This way every user should have access to it (ie before the researcher experience doesn't have a footer so the cookie was still rendering)